lib/TestPlanner.lua

```diff
@@ -49,9 +49,19 @@ end
 	as state doesn't need to be passed around between functions or explicitly
 	global.
 ]]
-function TestPlanner.createEnvironment(builder)
+function TestPlanner.createEnvironment(builder, optionalEnv)
```

Comment:
Is adding this environment to the planner enough to get access to these functions from _within_ an `it` block? There are two separate environments TestEZ injects and I don't remember if stuff we inject for test planning is usable at test run-time as well (inside an `it` block)

Reply:
We call getfenv on the module, which in the case of test is a function. So it sets this on the environment at the module level.

If we want it only at the "it" level, we can include it with the expect environment.